dashboard/app/models/course_version.rb

```diff
@@ -198,4 +198,8 @@ def summarize_for_unit_selector
       units: units.map(&:summarize_for_unit_selector).sort_by {|u| u[:position]}
     }
   end
+
+  def hoc?
+    course_offering&.category == 'hoc'
+  end
```

Comment:
@bethanyaconnor I'm trying to get this PR from before leave to a stopping point. Question: do you think it makes sense to jump ahead to using course_offering.category here, when the other helpers are using curriculum umbrella here? https://github.com/code-dot-org/code-dot-org/blob/c36aedfb061cdee27873e826caac32be80f11adf/dashboard/app/models/unit.rb#L892-L910

I'm reasonably confident that the behavior of this method will be identical in either case, so it's more a question of whether you'd rather have the new method conform to existing examples or blaze a trail toward maybe eliminating curriculum_umbrella in the future.

Reply:
We're actually discussing possibly adding a new field on course offerings to denote HoC courses! That said, I like the idea of using `course_offering.category` in the meantime and we can take the action item to update (if needed) in the future. It doesn't bother me that it's slightly different than the other methods

Reply:
Great! yeah, since you are planning to store that new flag on course offering, this seems like it builds in the right direction.